Library/Homebrew/extend/os/mac/caveats.rb

```diff
@@ -37,7 +37,9 @@ def plist_caveats
 
     if f.plist_manual || f.service?
       command = if f.service?
-        f.service.command.join(" ")
+        f.service.command
+         .map { |arg| (arg =~ /\s/) ? "'#{arg}'" : arg } # wrap multi-word arguments in quotes
+         .join(" ")
```

Comment:
I think making this multiline makes it a little easier to follow

```suggestion
         .map do |arg| 
           next arg unless arg.match?(/\s/)
           
           # quote multi-word arguments
           "'#{arg}'"
         end.join(" ")
```

Reply:
👍